README.md

```diff
@@ -298,6 +298,9 @@ _TODO: Simple guide to host error codes to be completed_
 
 ### Rollup
 
+> **❗⚠️❗ Warning**: <br>
+> These modifications will override the default behavior and configuration of `tsdx`. As such they can invalidate internal guarantees and assumptions. These types of changes can break internal behavior and can be very fragile against updates. Use with discretion!
```

Comment:
TSDX should be capitalized. We should be consistent.

Reply:
Ah good catch, I mix these up often. Fixed